l10n_pt_partner/README.rst

```diff
@@ -0,0 +1,65 @@
+Portugal - Partners
+============================================================
+
+This modules extends partners, suppliers, banks and company information in
+order to meet the legal and functional requirements of the Portuguese
+localization.
+
+Features:
+--------------
+
+ * Adds the *Alias* field to company-type partners. It can be a trade name, a
+   shorter name or whatever alternative company name.
```

Comment:
It's a fact that most of the times the field will have names that are
shorter than the 'main' name. But not always. Imagine your company
sells directly to each McDonalds restaurant. As they can be operated by
different companies, you could have:
NameAlias NameMcRest, SAMcDonald's LisboaRest-o-matic, SLMcDonald's MadridAlso pls note that I've previously worked with an ERP that has a
similar alias name field and found that people enjoy its versatility. 

-- 
Pedro Castro Silva
Sossia
917213709
A Seg, 30-11-2015 às 01:16 -0800, Daniel Reis escreveu:
> In l10n_pt_partner/README.rst:
> > @@ -0,0 +1,65 @@
> > +Portugal - Partners
> > +============================================================
> > +
> > +This modules extends partners, suppliers, banks and company
> information in
> > +order to meet the legal and functional requirements of the
> Portuguese
> > +localization.
> > +
> > +Features:
> > +--------------
> > +
> > + * Adds the *Alias* field to company-type partners. It can be a
> trade name, a
> > +   shorter name or whatever alternative company name.
> Why not name it "Short Name"?
> —
> Reply to this email directly or view it on GitHub.